public/data/smartCitiesData.json

```diff
@@ -119,7 +119,7 @@
       "peakDelay": 0.35,
       "accessToPublicTransport": 0.112,
       "grossParklandArea": 0.693,
-      "greenhouseGasEmissions1": 4,
+      "greenhouseGasEmissions1": 3.582318687,
```

Comment:
That spreadsheet behaviour is strange. I guess in the spreadsheet view of the world, whatever underlying data there is, export to CSV should match what you see on the screen as closely as possible...?